### ISSUE TITLE
'⚠️' breaks line rendering in certain contexts 

### ISSUE THREAD

**Issue by cxnder** [Vector35/binaryninja-api#3652]
**Version and Platform (required):**
Binary Ninja Version: 3.2.3834, e88d9c23
Platform: macOS Monterey (12.4) (x86_64)

**Bug Description:**
⚠️ breaks text rendering in certain contexts. We use this as an analysis tag in certain contexts, so it can be problematic/confusing.

This is a Qt-wide bug.

**Repro:**

### In BinaryNinja
part 1:
in python console: `print('⚠️')` -- breaks input text box, and output of this will render 3 blank lines, which if copied actually still contain '⚠️'.

part 2:
in python console: `log_alert('⚠️')` -- renders fine in the log view. displays malformed image in Alert popup.
This problem also occurs when clicking the tag in other contexts that would display it in a popup (tags).

### Qt, via pyside:
this repros the `log_alert` issue above. The python console issues likely occur from how we paint the broken character Qt displays.

bug.py:
```py
# This displays the same broken icon regardless of whether QPushButton, QTextEdit, or QLineEdit is used.
import sys
from PySide6.QtWidgets import QApplication, QPushButton
app = QApplication(sys.argv)
window = QPushButton("⚠️")
window.show()
app.exec_()
```
```
$ export PYTHONPATH=~/Qt/6.3.0/clang_64/pyside/site-packages/
$ python3 bug.py
```

**extra info:** 
Default macOS 12.4 fonts and theme being used.

**Comment by torarnv** [comment]
Possibly related to https://bugreports.qt.io/browse/QTBUG-108799

**Comment by psifertex** [comment]
Looks like it, thanks! Saves us the trouble of filing a dupe. :-)

**Comment by plafosse** [comment]
This has been fixed with a Qt update and a workaround.

**Comment by psifertex** [comment]
Actually, I think this is still an issue, re-opening until we upgrade to 6.4.3 or 6.5.0 qt.

**Comment by psifertex** [comment]
This is fixed upstream and is solely dependent on us upgrading to QT 6.4.3.

**Comment by D0ntPanic** [comment]
Fixed in 3.4.4191
